graph_tool/forms.py

```diff
@@ -2,12 +2,24 @@
 
 class UploadJSONandImage(forms.Form):
     file_field = forms.FileField(
-        # widget=forms.ClearableFileInput(attrs={'multiple': True})
-        error_messages={'required': 'Please upload json file and image files'}
+        widget=forms.ClearableFileInput(attrs={'multiple': True})
```

Comment:
what is widget variable for?

Reply:
to allow upload multiple files using the same field